etc/profile-m-z/torbrowser-launcher.profile

```diff
@@ -15,7 +15,8 @@ noblacklist ${HOME}/.local/share/torbrowser
 include allow-python2.inc
 include allow-python3.inc
 
-blacklist /opt
+noblacklist /opt/tor-browser
+blacklist /opt/*
```

Comment:
Why not just `whitelist /opt/tor-browser`.

Reply:
I didn't realize it would work as intended. Updated.